### PR TITLE
Support nodejs-mobile and fix CI

### PR DESCRIPTION
**Pull request by staltz**
Supersedes PR #22.

- [x] PR https://github.com/ssb-ngi-pointer/ssb-validate2-rsjs/pull/21 merged
- [x] Use a proper npm release of `-node`, not a git branch

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "author": "Andrew Reid <glyph@mycelial.technology>",
   "license": "AGPL-3.0",
   "dependencies": {
-    "ssb-validate2-rsjs-node": "~0.3.0",
+    "ssb-validate2-rsjs-node": "~0.4.0",
     "ssb-validate2-rsjs-wasm": "~0.3.0"
```
Comment (mycognosist) [inline package.json:14]: `-wasm` is now also at version `0.4.0`

**Review by mycognosist — APPROVED** [review]
Great! There's just the small dependency change I noted (`0.3.0` -> `0.4.0` for `-wasm`).

File: package.json (hunk 1 above)
Comment (staltz) [reply package.json:14]: It has to be published on npm, though, I'll do that 1st

Comment (mycognosist) [reply package.json:14]: Ah of course. Thank you 🙏🏻

**Pull request merged**